html-forms/html-forms/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form>
+      <p>Email</p>
+      <label for="email"></label>
+      <input type="email" name="email" id="email" />
+
+      <p>Why is coding cool?</p>
+      <label for="questionOne"></label>
+      <select name="why-coding-is-cool-select" id="questionOne">
+        <option selected>Select an option</option>
+        <option>Because it's interesting</option>
+        <option>Because it's fun</option>
+        <option>Because it's powerful</option>
+      </select>
+
+      <p>Favorite Language (choose one)</p>
+      <input type="radio" name="favoriteLanguage" id="HTML" value="HTML" />
+      <label for="HTML">HTML</label>
+      <input type="radio" name="favoriteLanguage" id="CSS" value="CSS" />
+      <label for="CSS">CSS</label>
+      <input
+        type="radio"
+        name="favoriteLanguage"
+        id="JavaScript"
+        value="JavaScript" />
+      <label for="JavaScript">JavaScript</label>
+
+      <p>Would you like to sign up for more information?</p>
+      <input type="checkbox" name="subscribe" id="questionTwo" value="yes" />
+      <label for="questionTwo">Sign me up!</label>
+
+      <p>Comments</p>
+      <label for="comments-textarea">
+        <textarea
+          name="comments"
+          id="comments-textarea"
+          cols="30"
+          rows="10"></textarea>
+      </label>
```

Comment:
You can definitely nest like this, but you still need to have some sort of text associated with the `label`. You can get rid of the `p` on line 42, and throw that text inside the `label`